test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -117,15 +143,79 @@ private void DoWorkspaceOpenAndCompare(string filePath)
 
             var newEngineCompileAndExecutionTime = model.EngineController.CompileAndExecutionTime;
 
-            Console.WriteLine("Compile and Execution time old Engine={0:0.0}+{1:0.0} ms, new Engine={2:0.0}+{3:0.0} ms",
+            Console.WriteLine("Compile and Execution time old Engine={0:0.0}+{1:0.0} ms, new Engine={2:0.0}+{3:0.0} ms, hardCoded={4:0.0}",
                 oldEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, oldEngineCompileAndExecutionTime.executionTime.TotalMilliseconds,
-                newEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, newEngineCompileAndExecutionTime.executionTime.TotalMilliseconds);
+                newEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, newEngineCompileAndExecutionTime.executionTime.TotalMilliseconds,
+                hardCodedExecutionTime.TotalMilliseconds);
             var execution = (Path.GetFileName(filePath),
                 oldEngineCompileAndExecutionTime.compileTime, oldEngineCompileAndExecutionTime.executionTime,
-                newEngineCompileAndExecutionTime.compileTime, newEngineCompileAndExecutionTime.executionTime);
+                newEngineCompileAndExecutionTime.compileTime, newEngineCompileAndExecutionTime.executionTime,
+                hardCodedExecutionTime);
             executionData.Add(execution);
         }
 
+        private void WarmUp()
+        {
+            if (!warmupDone)
+            {
+                var warmupFile = Path.Combine(TestDirectory, "core", "performance", "simple_point.dyn");
```

Comment:
so is this warmup basically just loading ASM?

Reply:
Yes, plus some other stuff related to core. I don't know yet if this is enough or if we need a more complicated graph.